dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -1,6 +1,5 @@
 import { css } from '@emotion/react';
```

Comment:
I noticed a lot of components that are part of the LiveBlock live here (in the dotcom-rendering subdirectory), is the intention to share them, or is it easier to keep them DCR specific?

Reply:
I suspect there'll be a couple more iterations on live blocks:

1. To lift more functionality into the container component, as @lucymonie mentioned above. The timestamp functionality is a good candidate for this, but I think probably the title and header can go that way too.
2. To strip out the logic for stretching content full width (`headerElement`, `typesWeStretch` etc.), since the designs no longer call for this.

Again, as Lucy mentioned above, it may be best to create follow-up tickets to handle this in small chunks?

Reply:
Yes, I think there are more pieces we could share, but I wouldn't think to make them part of the LiveBlockContainer, but potentially separate compose-able elements that could be children of it. Either way, I agree that these can be done as follow-up work.

Reply:
On the point about `typesWeStretch`, I _think_ we'll still need this for the pinned images / videos at the top of the blocks. I could be misunderstanding though.